spec/index.bs

```diff
@@ -5986,7 +5989,10 @@ A {{GPUBindGroupLayout}} object has the following internal slots:
                                 - |entry|.{{GPUBindGroupLayoutEntry/storageTexture}}.{{GPUStorageTextureBindingLayout/viewDimension}} is not
                                     {{GPUTextureViewDimension/"cube"}} or {{GPUTextureViewDimension/"cube-array"}}.
                                 - |entry|.{{GPUBindGroupLayoutEntry/storageTexture}}.{{GPUStorageTextureBindingLayout/format}} must be a format
-                                    which can support storage usage.
+                                    which can support storage usage for the given
+                                    |entry|.{{GPUBindGroupLayoutEntry/storageTexture}}.{{GPUStorageTextureBindingLayout/access}}
+                                    according to the [[#plain-color-formats|Plain color formats]] table.
```

Comment:
I clearly forgot my Bikeshed linking syntax, so thanks for cleaning that up. But also we could link it like so for consistency with the rest of the doc, which would include the header number. I leave that up to you.

```suggestion
                                    according to the [[#plain-color-formats]] table.
```